provisions/roles/scanner/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Install Atomic
+  yum: name=atomic state=present
+  sudo: yes
+
+- name: Get Docker installed
+  service: name=docker enabled=yes state=started
+  sudo: yes
+
+- name: Enable Docker registry
+  replace: >
+    dest=/etc/sysconfig/docker
+    regexp="^#?\s*ADD_REGISTRY=.*"
+    replace='ADD_REGISTRY="--add-registry {{ public_registry }}"'
+
+- name: Restart Docker
+  service: name=docker state=restarted enabled=yes
+
+- name: Get scanner image pulled
+  shell: docker pull registry.centos.org/pipeline-images/pipeline-scanner
+#  docker_image: 
+#    name: pipeline-images/pipeline-scanner
+#    tag: latest
+#    state: present
+
+- name: Run Atomic install
+  shell: atomic install registry.centos.org/pipeline-images/pipeline-scanner:latest
```

Comment:
Running `atomic install` would take care of pulling the Docker image as well. So, it can help us get rid of lines 19-20.